Docs/Specifics/Vision.md

```diff
@@ -2,64 +2,113 @@
-This is also the case for our robots—we have a camera connected to a coprocessor that lets us use visual input in our code.
+When we navigate the world, we have our eyes and the parts of our brain that process that visual input to help us.
+This is also the case for our robots—we have a camera connected to a processor that lets us use visual input in our code.
 
 We primarily have two uses for vision: target detection and pose estimation.
-The first usually involves some sort of visual fiducial marker (something that can act as a known point of reference that you can use to find your current location) near the target that we can align to and then score which is relatively easy to set up.
+The first usually involves some sort of visual fiducial marker (something that can act as a known point of reference to find your current location) near the target that we can align to and then score, which is relatively easy to set up.
 The second involves a more complicated system of identifying where certain visual fiducial markers are relative to the robot and then using that to generate an estimated pose of the robot on the field.
 
-### Retroreflective tape
+### AprilTags
 
-Retroreflective (RR) tape is the shiny stuff you see on roads, safety vests, traffic cones, and more.
-For most reflective surfaces, any light hitting it bounces off at an angle, but RR surfaces will reflect the majority of the light back at the source.
-This makes it a lot easier to accurately identify targets.
-For example, in Charged Up there was RR tape on the cone targets which can be used to align for scoring.
+AprilTags are a system of visual tags developed by researchers at the University of Michigan to provide low overhead, high accuracy localization for many different applications.
 
-### AprilTags
+![Robots at the University of Michigan with AprilTags](/Assets/Apriltags.jpg)
 
-Apriltags are a system of visual tags developed by researchers at the University of Michigan to provide low overhead, high accuracy localization for many different applications.
 They look similar to QR codes, but with less pixels.
-This trades amount of information in each tag for quick, easy, and robust detection.
-In the context of FRC, AprilTags are useful for helping your robot know where it is on the field, so it can align itself to some goal position.
-The idea behind this is that when the camera detects an Apriltag, it will calculate the distance, angle, etc. that the robot is from that target and then map that to a diagram that shows where all the Apriltags are on the field and then estimate where the robot is from there.
-This is useful because with a reliable pose estimation system we can automate more of the scoring process.
+This trades the amount of information in each tag for quick, easy, and robust detection.
+There are several different "families" of tags depending on how many bits a tag can represent.
```

Comment:
add a sentence about what the different types of families are for/how they differ. Something along the lines of "Different families have different sizes, shapes, and patterns of white, black, and "data" squares. This provides tradeoffs between detail, robustness, and number of potential IDs."